src/login_helper.js

```diff
@@ -61,6 +66,347 @@ function filterPastAndUpcomingTrips(travelerTrips) {
+  endDateInput.setAttribute("min", `${today}`);
+  endDateInput.setAttribute("max", "2023-12-31");
+  appendToRequestForm(endDateInput);
+}
+
+function prepareTripRequestData(destinations) {
+  let tripRequestData = [];
+  let tripRequestDestination = document.getElementById("destination").value;
+  let numberOfTravelers = document.getElementById("number-of-travs").value;
+  let tripRequestStartDate = document.getElementById("start").value;
+  let tripRequestEndDate = document.getElementById("end").value;
+  let i;
+  for (i = 0; i < destinations.length; i++) {
+    if (tripRequestDestination === destinations[i].destination) {
+      let destinationIdentifier = destinations[i].id;
+      tripRequestData.push(destinationIdentifier);
+    }
+  }
+  tripRequestData.push(
+    "Destination: " + tripRequestDestination,
+    "Number of Travelers: " + numberOfTravelers,
+    "Trip Start Date: " + tripRequestStartDate,
+    "Trip End Date:" + tripRequestEndDate
+  );
+  return tripRequestData;
+}
+function renderDestinationImage(destinationInfo) {
+  let destinationImage = destinationInfo.image;
+  var img = document.createElement("img");
+  img.className = "destination-image";
+  img.src = `${destinationImage}`;
+  document.querySelectorAll("section")[3].appendChild(img);
+}
+
+function getValuesFromInputs() {
+  let values = [];
+  let tripRequestDestination = document.getElementById("destination").value;
+  let tripRequestStartDate = document.getElementById("start").value;
+  let tripRequestEndDate = document.getElementById("end").value;
+  let numberOfTravelers = document.getElementById("number-of-travs").value;
+  values.push(
+    tripRequestDestination,
+    tripRequestStartDate,
+    tripRequestEndDate,
+    numberOfTravelers
+  );
+  return values;
+}
+
+function calculateTripRequestCosts(destination) {
+  let costs = [];
+  let values = getValuesFromInputs();
+  let numberOfTravelers = values[3];
+  let tripRequestStartDate = values[1];
+  let tripRequestEndDate = values[2];
+  let flightCosts =
+    numberOfTravelers * destination.estimatedFlightCostPerPerson;
+  let lodgingCostsPerDay =
+    numberOfTravelers * destination.estimatedLodgingCostPerDay;
+  let totalLodgingCosts =
+    lodgingCostsPerDay * daysBetween(tripRequestStartDate, tripRequestEndDate);
+  let costBeforeAgentFee = flightCosts + totalLodgingCosts;
+  let agentFee = costBeforeAgentFee * 0.1;
+  let totalTripCost = costBeforeAgentFee + agentFee;
+  costs.push(
+    flightCosts,
+    lodgingCostsPerDay,
+    totalLodgingCosts,
+    costBeforeAgentFee,
+    agentFee,
+    totalTripCost
+  );
+  return costs;
+}
+
+function renderDestinationName(destination) {
+  let destinationName = document.createTextNode(
+    `Trip Destination: ${destination.destination}`
+  );
+  let destinationNameParagraph = document.createElement("paragraph");
+  destinationNameParagraph.appendChild(destinationName);
+  document.querySelectorAll("section")[3].appendChild(destinationNameParagraph);
+}
```

Comment:
All these methods look good, but is this really appropriate for the login helper?  Maybe we should split these files out in to appropriate sub helper files...an agent dashboard helper, travler dashboard helper, login helper, etc. 

Organize the files in a more sensible manner :)

Reply:
more well named smaller files is better than less long messy files